server/bootstrap/src/main/java/org/kaaproject/kaa/server/bootstrap/BootstrapServerApplication.java

```diff
@@ -37,6 +42,13 @@
      *            the arguments
      */
     public static void main(String[] args) {
+        Thread.setDefaultUncaughtExceptionHandler(new Thread.UncaughtExceptionHandler() {
+            @Override
+            public void uncaughtException(Thread th, Throwable ex) {
+                LOG.error("Uncaught exception: ", ex);
```

Comment:
please log thread id and/or name. Smth like "Thread [{}] uncaught exception: {}", ex.getMessage, ex);